src/main/kotlin/nl/info/zac/app/note/model/RestNote.kt

```diff
@@ -0,0 +1,45 @@
+/*
+ * SPDX-FileCopyrightText: 2021 Atos, 2025 INFO.nl
+ * SPDX-License-Identifier: EUPL-1.2+
+ */
+package nl.info.zac.app.note.model
+
+import jakarta.json.bind.annotation.JsonbProperty
+import nl.info.zac.note.model.Note
+import nl.info.zac.util.AllOpen
+import nl.info.zac.util.NoArgConstructor
+import java.time.ZonedDateTime
+import java.util.UUID
+
+@NoArgConstructor
+@AllOpen
+data class RestNote(
+    var id: Long?,
+    var zaakUUID: UUID,
+
+    @field:JsonbProperty("tekst")
+    var text: String,
+
+    /**
+     * Date-time when the note was last modified.
+     */
+    @field:JsonbProperty("tijdstipLaatsteWijziging")
+    var dateTimeLastModified: ZonedDateTime?,
```

Comment:
Can this ever be null? If a note is created is should have a modified date-time I think

Reply:
yes, it is null when a note is created by the frontend. There are loads of improvements that can be done here still but I first wanted to convert the code to Kotlin. :-)

Reply:
the issue with these REST data classes is that currently they are used both by the getters as well as the setters so to speak. Hence the nullable fields in case of the setters. We should separate these in two. Maybe a future PR..